### ISSUE TITLE
Unusual downtime

### ISSUE THREAD

**Issue by urticadioica** [igrigorik/vimgolf#146]
vimgolf.com is down a lot. It's been that way ever since I first came here, a couple years ago. Typically, it seems like the site will slow down, then several minutes later go down completely with Heroku's famous Application Error page. It will usually take maybe 15 seconds to load the Application Error. The downtime normally lasts maybe 10-20 minutes, and on average, I believe it happens a couple times a day.

Again, that's the *typical* pattern I've learned to love over the years. I don't think vimgolf.com has been up at any time in the last 24 hours. The first several hours were "We've been notified about this issue and we'll take a look at it shortly.", and I've got an instant Application Error since then -- no 15 second delay.

I and most players I've talked to have learned to laugh off the typical downtime. But this is out of the ordinary.

**Comment by igrigorik** [comment]
This is my fault.

- I've recently migrated VimGolf to a new DB provider (more resources, etc).. ironically, in the hope of alleviating downtime.
- It looks like I've misconfigured the replica & failover logic and that's exactly what happened and left the app unable to connect to the DB. 

I've updated the config but will investigate further over the weekend.. Having trouble testing the replica failover logic at the moment.
